### PR TITLE
Kingfisher Replica and Document Ubuntu Pro

### PR DESCRIPTION
**Pull request by RobHooper**

### CODE REVIEW

File: docs/deploy/ubuntu_pro.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+`Ubuntu Pro <https://ubuntu.com/pro>`__ is an enterprise subscription from Canonical Ltd that provides (among other features) extended security support after the standard end-of-life date, this is also referred to as Extended Security Maintenance (ESM).
+
+.. note::
+
+   `Salt Stack does not officially support ESM<https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-supported-operating-systems.html#ubuntu>`__, an end-of-life server will need removing from the Deploy repo regardless.
+
+Installation
+------------
+
+#. Purchase an Ubuntu Pro licence from `Canonical <https://ubuntu.com/pro/subscribe>`__.
```
Comment (RobHooper) [inline docs/deploy/ubuntu_pro.rst:13]: This line may need more detail around the options we selected.

Comment (RobHooper) [inline docs/deploy/ubuntu_pro.rst:13]: We cannot purchase individual licences for VPS servers (Canonical only provide these in bulk), so this process is only for hardware servers.

File: docs/deploy/ubuntu_pro.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Ubuntu Pro
+==========
+
+`Ubuntu Pro <https://ubuntu.com/pro>`__ is an enterprise subscription from Canonical Ltd that provides (among other features) extended security support after the standard end-of-life date, this is also referred to as Extended Security Maintenance (ESM).
+
+.. note::
+
+   `Salt Stack does not officially support ESM<https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-supported-operating-systems.html#ubuntu>`__, an end-of-life server will need removing from the Deploy repo regardless.
```
Comment (jpmckinney) [inline docs/deploy/ubuntu_pro.rst:8]:
```suggestion
   `Salt does not support ESM <https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-supported-operating-systems.html#ubuntu>`__.
```

File: docs/deploy/ubuntu_pro.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+#. Purchase an Ubuntu Pro licence from `Canonical <https://ubuntu.com/pro/subscribe>`__.
+#. :doc:`SSH<../use/ssh>` into the server as the ``root`` user.
+#. Install Ubuntu Pro software.
+
+   .. code-block:: bash
+
+      apt install ubuntu-advantage-tools
+
+#. Attach licence key and configure.
```
Comment (jpmckinney) [inline docs/deploy/ubuntu_pro.rst:21]:
```suggestion
#. Attach the licence key.
```

File: docs/deploy/ubuntu_pro.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+.. note::
+
+   `Salt Stack does not officially support ESM<https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-supported-operating-systems.html#ubuntu>`__, an end-of-life server will need removing from the Deploy repo regardless.
+
+Installation
+------------
+
+#. Purchase an Ubuntu Pro licence from `Canonical <https://ubuntu.com/pro/subscribe>`__.
+#. :doc:`SSH<../use/ssh>` into the server as the ``root`` user.
+#. Install Ubuntu Pro software.
```
Comment (jpmckinney) [inline docs/deploy/ubuntu_pro.rst:15]:
```suggestion
#. Install the Ubuntu Pro Client.
```

File: docs/deploy/ubuntu_pro.rst [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+   .. code-block:: bash
+
+      apt install ubuntu-advantage-tools
+
+#. Attach licence key and configure.
+
+   .. code-block:: bash
+
+      pro attach <TOKEN>
```
Comment (jpmckinney) [inline docs/deploy/ubuntu_pro.rst:25]:
```suggestion
      pro attach TOKEN
```

File: docs/deploy/ubuntu_pro.rst [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+      pro config set apt_news=false
```
Comment (jpmckinney) [inline docs/deploy/ubuntu_pro.rst:26]:
```suggestion

#. Disable advertisements:

   .. code-block:: bash

      pro config set apt_news=false
```

**Review by jpmckinney — APPROVED** [review]

**Comment by RobHooper** [comment]
Re-added prometheus monitoring since the server is staying online and we need monitoring.

**Pull request merged**